### ISSUE TITLE
[BITV] 9.1.4.3/6.2 - The "Edit your Profile visibility" link does not meet the contrast requirements for text - it is 2.4:1. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36958]
- [ ] Fix
- [ ] Backport

![ea791921f8d91989b62585d40226bf0f](https://user-images.githubusercontent.com/1315170/222224816-d682b122-68dd-4917-9556-5375cece20ef.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6fe46b78da-v6-n2

</details>
